### PR TITLE
[Editor] Persistent value

### PR DESCRIPTION
**Pull request by fructozer**
Support save and load value to account's persistent value

### CODE REVIEW

File: src/api/components/components.ts [hunk 1]
```diff
@@ -4123,6 +4176,7 @@ export const initComponents = () => {
 		VALUE_COPY:          { name: 'Value Copy', component: ValueCopyMechanic },
 		VALUE_DISTANCE:      { name: 'Value Distance', component: ValueDistanceMechanic },
 		VALUE_HEALTH:        { name: 'Value Health', component: ValueHealthMechanic },
+		VALUE_LOAD:          { name: 'Value Lore', component: ValueLoadMechanic },
```
Comment (neYaANK) [inline src/api/components/components.ts:4179]: You forgot to rename it :)

Comment (fructozer) [reply src/api/components/components.ts:4179]: Damn it

File: src/api/components/components.ts [hunk 2]
```diff
@@ -3586,9 +3597,32 @@ class ValueLocationMechanic extends ProMechanic {
...
+		}, false);
+	}
+
+	public static override new = () => new this();
+}
+
+class ValueLoadMechanic extends ProMechanic {
+	public constructor() {
+		super({
+			name:         'Value Location',
```
Comment (Travja) [inline src/api/components/components.ts:3615]: Need to rename it here too

File: src/api/components/components.ts [hunk 3]
```diff
@@ -3586,9 +3597,32 @@ class ValueLocationMechanic extends ProMechanic {
...
+			description:  'If there is a value already stored on the account, that value will be retrieved and ban be used as a normal value',
```
Comment (Travja) [inline src/api/components/components.ts:3616]: I assume this should say "…and then be used…"

File: src/api/components/components.ts [hunk 4]
```diff
@@ -3703,16 +3752,17 @@ class ValueRandomMechanic extends ProMechanic {
 			data:         [
 				new StringSelect('Key', 'key', 'value')
 					.setTooltip('The unique key to store the value under. This key can be used in place of attribute values to use the stored value'),
-				new BooleanSelect('Integer', 'integer', false)
```
Comment (Travja) [inline src/api/components/components.ts:3706]: It's there a reason this option was removed?

**Review by Travja — CHANGES_REQUESTED** [review]

File: src/api/components/components.ts (hunk 4 above)
Comment (fructozer) [reply src/api/components/components.ts]: Maybe I forgot it when I merged the branch

**Review by Travja — APPROVED** [review]

**Review by Travja — APPROVED** [review]

**Pull request merged**